openstack/blockstorage/v1/volumes/requests_test.go

```diff
@@ -3,6 +3,7 @@ package volumes
 import (
 	"testing"
```

Comment:
@saad-ali I didn't move those cause they were explicitly named test files and were not affecting the production code. I agree that it will probably be cleaner to have all of the test files in the package, however there is some code that is imported within the package, such between urls.go and urls_test.go. So in order to move that it would require altering the scope of the functions in urls.go.